.github/ISSUE_TEMPLATE/flaky_test_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Flake Report
+about: Report a flaky test in KubeVirt
+title: 'test_name'
+labels: kind/flake
+assignees: ''
+---
+
+# What happened
+
+<!-- insert test name -->
+Flaky test detected: {test_name} [1]
+
+<!-- labels for flaky tests -->
+/kind flake
+/priority critical-urgent
```

Comment:
I guess this should only be added if it is actually urgent?

Reply:
I think any flake deserves the urgent label, since it harms our CI.

Reply:
Isn't this better fit for PR template?

Reply:
Right I think issues are good for backtracking but the label `/priority critical-urgent` doesn't have  a meaning for issue, correct?

Reply:
For issues this is just a marker and doesn't do anything besides that, you are right. However, I think that this still has meaning.